src/component/archive/ArchiveTable/ReviveArchivedFeatureCell/ReviveArchivedFeatureCell.tsx

```diff
@@ -0,0 +1,20 @@
+import { VFC } from 'react';
+import { ActionCell } from '../../../common/Table/cells/ActionCell/ActionCell';
+import { Undo } from '@mui/icons-material';
+import { IconButton } from '@mui/material';
+
+interface IReviveArchivedFeatureCell {
+    onRevive: any;
+}
+
+export const ReviveArchivedFeatureCell: VFC<IReviveArchivedFeatureCell> = ({
+    onRevive,
+}) => {
+    return (
+        <ActionCell>
+            <IconButton onClick={onRevive}>
```

Comment:
This needs to be a `<PermissionIconButton />` and take the projectId and permission as props.